compute/oslogin/service_account_ssh_test.py

```diff
@@ -84,19 +84,20 @@ def test_main(capsys):
         'oslogin', 'v1', cache_discovery=False, credentials=credentials)
     account = 'users/' + account_email
 
-    @retry(wait_exponential_multiplier=1000, wait_exponential_max=300000,
-           stop_max_attempt_number=10)
+    # Multiple exception types can be raised, using generic Exception as a catch all.
+    # I was not able to track all the exception types that can be raised from this block
+    # of code.
+    @backoff.on_exception(backoff.expo,
+                      Exception,
```

Comment:
could consider a tuple of multiple exceptions as an alternative to the generic

Reply:
Done! Changed to include ones I could find for now, and additional ones could be easily added in the future.